schema/pet-insurance.json

```diff
@@ -288,25 +289,20 @@
             "minItems": 1,
             "maxItems": 3,
             "items": {
-                "$ref": "#/domains/PetInsurance/types/VoluntaryExcessItem"
+                "$ref": "#/domains/PetInsurance/types/VoluntaryExcess"
             }
         },
-        "VoluntaryExcessItem": {
+        "VoluntaryExcess": {
             "type": "object",
             "description": "Flight fare information.",

```

Comment:
This (and a number of other definitions which mention passengers) are most likely a copy-paste leftovers. Please fix

Reply:
![image](https://user-images.githubusercontent.com/34474056/54904250-b2c1ca00-4ee6-11e9-9beb-8ad8045e89a3.png)

I don't understand. What should I modify? Those red-lines are the lines before modification. They are not in code anymore.

Reply:
Do you mean the description?

Reply:
Yes, "flight fare information". Also search the same number for "passenger" text.